app/src/main/java/com/gut/follower/RecordFragment.java

```diff
@@ -150,29 +165,46 @@ public void onFailure(Call<Track> call, Throwable t) {
     public void onMapReady(GoogleMap googleMap) {
         map = googleMap;
         map.setMyLocationEnabled(true);
-        LocationManager locationManager = (LocationManager) this.getActivity().getSystemService(Context.LOCATION_SERVICE);
+
+        LocationManager locationManager =
+                (LocationManager) this.getActivity().getSystemService(Context.LOCATION_SERVICE);
         Criteria criteria = new Criteria();
-        Location location = locationManager.getLastKnownLocation(locationManager.getBestProvider(criteria, false));
+        Location location =
+                locationManager
+                        .getLastKnownLocation(locationManager.getBestProvider(criteria, false));
+
         if(location != null){
-            map.animateCamera(CameraUpdateFactory.newLatLngZoom(new LatLng(location.getLatitude(), location.getLongitude()), 13));
+            map.animateCamera(
+                    CameraUpdateFactory
+                            .newLatLngZoom(new LatLng(location.getLatitude(),
+                                                      location.getLongitude()),
+                                                      13));
 
             CameraPosition cameraPosition = new CameraPosition.Builder()
-                    .target(new LatLng(location.getLatitude(), location.getLongitude()))      // Sets the center of the map to location user
-                    .zoom(13)                   // Sets the zoom
-                    .build();                   // Creates a CameraPosition from the builder
+                    // Sets the center of the map to location user
+                    .target(new LatLng(location.getLatitude(), location.getLongitude()))
+                    .zoom(13) // Sets the zoom
+                    .build(); // Creates a CameraPosition from the builder
             map.animateCamera(CameraUpdateFactory.newCameraPosition(cameraPosition));
         }
     }
 
     private class GpsProvider implements LocationListener{
-
+    // TODO: This class should be separated.
```

Comment:
I agree, but I didn't know how to change the UI from a separate class, so I did it like this for now.